vale/dictionary/p.jsonnet

```
@@ -1,5 +1,6 @@
 local word = import './word.jsonnet';
 [
+  word.new('PagerDuty', '', 'noun') { product: true },
```

Catch the common misspellings:

```suggestion
  word.new('PagerDuty', '', 'noun') { description: 'https://www.pagerduty.com/', product: true, swaps: { 'pager[dD]uty': 'PagerDuty', Pagerduty: 'PagerDuty', } },
```